src/bridge/SequencerInbox.sol

```diff
@@ -141,10 +141,11 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
     bool internal immutable hostChainIsArbitrum = ArbitrumChecker.runningOnArbitrum();
     // True if the chain this SequencerInbox is deployed on uses custom fee token
     bool public immutable isUsingFeeToken;
+    IEspressoTEEVerifier public espressoTEEVerifier;
```

Comment:
Should we put our variables at the end of all of the state variables and add a variable gap in here to make future upgrades easier?

Reply:
I think we need to keep the variables in the same position as we have in v2.1.3, I dont think we can change that as far as I know. Let me know if thats incorrect!
The proxy's are still remaining the same